packages/runtime/datastore/src/localChannelContext.ts

```diff
@@ -156,10 +159,6 @@ export class LocalChannelContext implements IChannelContext {
         this.channel = channel;
         this._isLoaded = true;
```

Comment:
Since you are touching this file, any chance you can remove this_isLoaded variable in favor of this implementaiton:
```
    public get isLoaded(): boolean {
        return this.channel !== undefined;
    }
```

Reply:
Changed